script.js

```diff
@@ -1,200 +1,82 @@
-//                             kgOutput.innerHTML = weight*0.453592;
-//                             event.stopPropagation(); 
-//                             } else 
-//                                 if (weightType.value === "Ounces") {
-//                                     output.style.visibility = 'visible';
-//                                     gramsOutput.innerHTML = weight*28.3495; 
-//                                     lbsOutput.innerHTML = weight*0.0625;
-//                                     ozOutput.innerHTML =  " ";
-//                                     kgOutput.innerHTML = weight*0.0283495;
-//                                     event.stopPropagation();
-//                                 }
-// };
-
-// second version of calculation function; fixes NaN output issue
-// const calculateTwo = () => {
-//       let weight = input.value;
-//         if (input.value === "") {
-//               output.style.visibility = 'hidden';
-//           } else
-//             if (weightType.value === "Kilograms") {
-//                       output.style.visibility = 'visible';
-//                       gramsOutput.innerHTML = weight*1000; 
-//                       lbsOutput.innerHTML = weight*2.20462;
-//                       ozOutput.innerHTML = weight*35.274;
-//                       kgOutput.innerHTML = weight*1;
-//                       } else 
-//                           if (weightType.value === "Grams") {
-//                               output.style.visibility = 'visible';
-//                               gramsOutput.innerHTML = weight*1; 
-//                               lbsOutput.innerHTML = weight*0.00220462;
-//                               ozOutput.innerHTML = weight*0.035274;
-//                               kgOutput.innerHTML = weight*0.001;
-//                               } else 
-//                                   if (weightType.value === "Pounds") {
-//                                       output.style.visibility = 'visible';
-//                                       gramsOutput.innerHTML = weight*453.592; 
-//                                       lbsOutput.innerHTML = weight*1;
-//                                       ozOutput.innerHTML = weight*16;
-//                                       kgOutput.innerHTML = weight*0.453592;
-//                                       } else 
-//                                           if (weightType.value === "Ounces") {
-//                                               output.style.visibility = 'visible';
-//                                               gramsOutput.innerHTML = weight*28.3495; 
-//                                               lbsOutput.innerHTML = weight*0.0625;
-//                                               ozOutput.innerHTML =  weight*1;
-//                                               kgOutput.innerHTML = weight*0.0283495;
-//                                           }
-//   };
+window.onload = toggleOutputDisplay();

```

Comment:
🤔 what does this do?

Reply:
It's supposed to keep the output field hidden when `inputEl = ""`.

> Theres only one tiny issue left. The window.onload looks weird can you tell me why?

It's not an event listener like the other three? Perhaps I should have gone with `window.addEventListener("load", function(event) { ...method }`.

Reply:
it should be `window.onload = toggleOutputDisplay` you want to assign `toggleOutputDisplay ` as the onload function not assign the output of `toggleOutputDisplay()`. One super simple alternative is to set 'hidden' class on the markup itself as default so you don't need to do any onload actions

Reply:
I went with your proposed alternative. Let me know if we can merge the branches after this push.